src/core_plugins/console/public/src/input.js

```diff
@@ -1,20 +1,20 @@
-let $ = require('jquery');
+const $ = require('jquery');
 require('ace');
-require('ace/ext-searchbox');
-let Autocomplete = require('./autocomplete');
-let SenseEditor = require('./sense_editor/editor');
-let settings = require('./settings');
-let utils = require('./utils');
-let es = require('./es');
-let history = require('./history');
+require('brace/ext/searchbox');
```

Comment:
again mixing `ace` and `brace`

Reply:
import ace ... is brace.

Reply:
Ah, I see, `import 'ace'` loads the ace webpack shim which loads brace.